providers.tf

```diff
@@ -0,0 +1,3 @@
+# Default provider information.
```

Comment:
Is this necessary in this case, where there is only a single provider?  @dav3r or @mcdonnnj?

Reply:
Since this is part of CyHy and will have to migrate to the COOL at some point, we may as well have a `providers.tf` for the eventual migration.

This should mirror what was previously in `main.tf` however: https://github.com/cisagov/assessment-data-import-terraform/blob/6c3ec84993dcd1c95b36b93a184bc40654267320/main.tf#L2-L4

Reply:
See @mcdonnnj's [later comment](https://github.com/cisagov/assessment-data-import-terraform/pull/4#discussion_r483104270).